sig-network/charter.md

```diff
@@ -0,0 +1,55 @@
+# SIG Network Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Network is responsible for the components, interfaces, and APIs which provide networking capabilities to Kubernetes
+users and workloads. SIG Network's goal is to provide simple, flexible abstractions on top of raw networking details, and to
+ensure Kubernetes provides the necessary networking features to enable application developers and cluster operators can
+meet their operational requirements.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Integration points with networking implementations (e.g. CNI integration)
+- Services / kube-proxy
```

Comment:
Maybe it's worth putting a sentence for each one that describes what these terms mean?

Services:
- APIs for defining and grouping networking endpoints
- APIs and implementation for load balancing (L3/4 (kube-proxy) + L7 (Ingress))